proposals/2021-05-gravity-bridge-deployment/README.md

```diff
@@ -0,0 +1,174 @@
+- The Gravity Cosmos module of Gaia
+- The orchestrator program which is run by Cosmos validators alongside the Gravity module
+- A market of relayers, who compete to submit transactions to Ethereum
+
+### Gravity.sol
+
+The Gravity Ethereum contract is a highly compact and efficient representation of weighted powers voting on Ethereum. It contains an Etheruem key from each Cosmos validator, as well as their voting power. This signer set is continuously updated as validation power changes on Cosmos, ensuring that it matches the current Cosmos validator set.
+
+Sending tokens, or updating the validator set, contained in Gravity.sol requires more than 66% of the total voting power to approve the action. In this way Gravity.sol mirrors Tendermint consensus on the Cosmos chain as closely as possible on Ethereum.
+
+### Gravity Cosmos Module
+
+The Gravity module governs and coordinates the bridge. Generating messages for the validators to sign with their Ethereum keys and providing these signatures to relayers who assemble and submit them to the Ethereum chain.
+
+### Orchestrator
+
+The Gravity bridge orchestrator performs all the external tasks the Gravity bridge requires for validators, which includes submission of signatures and submission of Ethereum events.
+
+While the Gravity module concerns itself with the correctness and consensus state of the bridge, the Orchestrator is responsible for locating and creating the correct inputs.
+
+### Market of Relayers
+
+Relayers are an unpermissioned role that observes the Cosmos chain for messages ready to be submitted to Ethereum.
+
+The relayer then packages the validators signatures into an Ethereum transaction and submits that transaction to the Ethereum blockchain. All rewards in the Gravity bridge design are paid to msg.sender on Ethereum. This means that relayers do not require any balance on the Cosmos side and can immediately liquidate their earnings into ETH while continuing to relay newer messages.
+
+## Security assumptions
+
+The Gravity bridge is designed with the assumption that the total amount of funds in Gravity.sol is less than the value of the validator set’s total staked tokens.
+
+If this assumption does not hold true, it would be more profitable for validators to steal the funds in the bridge and simply lose their stake to slashing.
+
+There is no automated enforcement of this assumption. It is up to the $ATOM holders to take action if the amount deposited in the bridge exceeds the total value of all stake on the hub.
+
+It should be noted that this condition is not unique to the Gravity bridge. The same dynamic exists for any IBC connection, and even exists in scenarios other than cross-chain communication. For example, in a hypothetical blockchain keeping domain name records, this same vulnerability would exist if the potential profit from exploiting the domain name system was greater than the value of the validator set’s total staked tokens.
```

Comment:
It would be great to include some description of how Gravity compares to IBC and why it was not possible or otherwise ideal to use IBC directly

Reply:
I've added a section on why we can not use IBC to bridge to Ethereum, a section comparing IBC and Gravity was previously attempted but removed due to the amount of technical nuance. I'd be interested in what properties you think are the most important to compare?